packages/@okta/vuepress-site/.vuepress/scripts/build-overview-pages.js

```diff
@@ -37,36 +41,44 @@ function generatedLinks(arr, parent = null) {
   if (arr) {
     for (let el of arr) {
       if (el) {
-        if (!el.path && !el.guideName) {
-          const parentTitle = sanitizeTitle(parent);
-          let path = '';
-          if (parentTitle !== 'guides' && parent.path) {
+        if (!el.path && parent && parent.path) {
+          const isGuide = !el.guideName && !parent.path.includes('docs/journeys');
+          const isJourney = !el.journeyName && parent.path.includes('docs/journeys');
+
+          if (isGuide || isJourney) {
```

Comment:
maybe it would be more straightforward if we inverse the check:
```
if (el.guideName || el. journeyName) {
   path = parent.path + sanitizeTitle(el) + "/";
} else {
   // all other checks
}
```
the more I look into this logic the more fragile it feels...